app/src/main/java/org/pragyan/dalal18/adapter/CompanyRecyclerAdapter.java

```diff
@@ -48,7 +48,14 @@ public void onBindViewHolder(@NonNull CompanyRecyclerAdapter.PortfolioViewHolder
 
         CompanyDetails currentCompanyDetails = list.get(position);
 
-        holder.companyNameTextView.setText(currentCompanyDetails.getCompanyName());
+        if (currentCompanyDetails.getGivesDividend()){
+        holder.companyNameTextView.setText(currentCompanyDetails.getCompanyName()+ context.getResources().getString(R.string.dividendSuffix));}
+        else if (currentCompanyDetails.isBankrupt()){
+            holder.companyNameTextView.setText(currentCompanyDetails.getCompanyName()+context.getResources().getString(R.string.bankruptSuffix));
```

Comment:
Do not concatenate text displayed with setText. Use resource string with placeholders.

When calling TextView#setText:

Never call Number#toString() to format numbers; it will not handle fraction separators and locale-specific digits properly. Consider using String#format with proper format specifications (%d or %f) instead.
Do not pass a string literal (e.g. "Hello") to display text. Hardcoded text can not be properly translated to other languages. Consider using Android resource strings instead.
Do not build messages by concatenating text chunks. Such messages can not be properly translated.

Warnings are there for a reason!